### PR TITLE
Tedefo 1023 multipe variables

### PR DESCRIPTION
**Pull request by rousso**

### CODE REVIEW

**Review by bertrand-lorentz — APPROVED** [review]
See my comment on the use of UnsupportedOperationException 

If it is indeed the intended behavior, you can merge this.

File: src/main/java/eu/europa/ted/efx/xpath/XPathScriptGenerator.java [hunk 1]
```diff
@@ -161,15 +163,27 @@ public BooleanExpression composePatternMatchCondition(StringExpression expressio
   @Override
   public <T extends Expression> BooleanExpression composeAllSatisfy(ListExpression<T> list,
       String variableName, BooleanExpression booleanExpression) {
+    throw new UnsupportedOperationException("Deprecated in SDK 0.8.0");
```
[the remark this answers is not in the document]
Comment (rousso) [reply src/main/java/eu/europa/ted/efx/xpath/XPathScriptGenerator.java:166]: Yes you are right. I should not have done that.
I did a similar thing on the EfxExpressionTranslator08 and then I did not notice that the ScriptGenerator is shared across versions.

[the remark this answers is not in the document]
Comment (rousso) [reply src/main/java/eu/europa/ted/efx/xpath/XPathScriptGenerator.java:166]: I will take care of it.

**Review by bertrand-lorentz — APPROVED** [review]
Looks good now.

**Pull request merged**